src/variant.rs

```diff
@@ -174,10 +250,25 @@ impl Variant {
         ))
     }
 
+    /// Creates a new dictionary entry GVariant.
+    pub fn new_dict_entry(key: &Variant, value: &Variant) -> Self {
```

Comment:
What would one do with the return value, how would it be inserted into a dict? Might be worth adding a short doc example :)

Reply:
In "GVariant" format (unlike D-Bus), a dict-entry can be used on its own and doesn't need to be in a dict. I'm just provide a binding for the C API, which doesn't document it's use either. I'll add a bit of info. :)